### PR TITLE
Add label to genetic-algorithm-files

### PR DESCRIPTION
**Pull request by Glitchy-Tozier**
Additionally, mention all the optimization-methods.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,4 +156,4 @@ The project includes several binaries within the `evolve_keyboard_layout` crate:
 The binaries rely on three library crates providing relevant data structures and algorithms:
 1. `keyboard_layout` - Provides a representation of keys, keyboards, and layouts and a layout generator that generates layout objects from given strings.
 1. `layout_evaluation` - Provides functionalities for reading, generating, and processing ngram data and datastructures and traits for evaluating several metrics.
-2. `layout_optimization` - Provides a connection to the genevo optimization algorithms by implementing a specialized genetic algorithm based on the evaluator in `layout_evaluation`.
+1. `layout_optimization` - Provides a connection to the genevo optimization algorithms by implementing a specialized genetic algorithm based on the evaluator in `layout_evaluation`.
```
Comment (Glitchy-Tozier) [inline README.md:159]: @dariogoetz This sentence/paragraph should probably be rewritten. Could you supply the text, as you're more knowledgeable of the structure of this project?

**Comment by Glitchy-Tozier** [comment]
@dariogoetz Do you know why the `Resolve conflicts`-button is disabled?

**Comment by dariogoetz** [comment]
> @dariogoetz Do you know why the `Resolve conflicts`-button is disabled?

I did some refactoring regarding the "rearrange folders" issue. This seems to have made things complicated. I'll try to resolve.

**Comment by dariogoetz** [comment]
The failing tests are due to the `Rocket` submodule having been moved (and some mistakes I made). I don't know, why it still looks for the submodule in the wrong location.
If the pull request is ready for merge, I will merge it unregarding these errors.

**Comment by Glitchy-Tozier** [comment]
@dariogoetz Ready to merge!

**Pull request merged**